internal/module/module.go

```diff
@@ -323,13 +362,12 @@ func sortItems(tfmodule *tfconf.Module, sortby *SortBy) {
 
 	if sortby.Name || sortby.Type {
 		sort.Sort(outputsSortedByName(tfmodule.Outputs))
-	} else {
-		sort.Sort(outputsSortedByPosition(tfmodule.Outputs))
-	}
-
-	if sortby.Name || sortby.Type {
 		sort.Sort(providersSortedByName(tfmodule.Providers))
 	} else {
+		sort.Sort(outputsSortedByPosition(tfmodule.Outputs))
 		sort.Sort(providersSortedByPosition(tfmodule.Providers))
 	}
+	// Always sort resources
```

Comment:
Nit: Can you add an empty line before this comment and remove the trailing line after `sort.Sort`?